packages/search/src/features/registration/deduplicate/service.ts

```diff
@@ -37,3 +49,180 @@ const extractRelatesToIDs = (bundle: fhir.Bundle) => {
       item.targetReference.reference.replace('Composition/', '')
   )
 }
+
+/**
+ * @author
+ * @kind module
+ * @description search for duplicate birth declarations
+ * @param { IBirthCompositionBody } body -  Params
+ * @param client
+ * @returns { Promise<ApiResponse<ISearchResponse<IBirthCompositionBody>, Context>> } -  Promise
+ * **/
+export const searchForDuplicates = async (
```

Comment:
great stuff remembering documentation! could we remove the self-documenting parts though; I think TypeScript handles `@param` and `@returns`. Also `@author` and `@kind` are redundant